mmdet/apis/ote/apis/detection/ote_utils.py

```diff
@@ -100,7 +101,19 @@ def __init__(self, update_progress_callback: UpdateProgressCallback):
 
     def on_train_batch_end(self, batch, logs=None):
         super().on_train_batch_end(batch, logs)
-        self.update_progress_callback(self.get_progress())
+        self.update_progress_callback(self.get_progress(), logs)
+
+    def on_epoch_end(self, epoch, logs=None):
+        self.past_epoch_duration.append(time.time() - self.start_epoch_time)
+        self.__calculate_average_epoch()
+        self.update_progress_callback(self.get_progress(), logs)
```

Comment:
Second argument for the `UpdateProgressCallback` should be of type `Optional[float]`. Is `logs` of a proper type?

Reply:
Thank you for your informative comments. I updated it to pass 'float'-type number instead of logs.